objects/src/transaction/inputs.rs

```diff
@@ -398,10 +393,19 @@ impl InputNote {
 }
 
 /// Returns true if this note belongs to the note tree of the specified block.
-fn is_in_block(note: &Note, proof: &NoteInclusionProof, block_header: &BlockHeader) -> bool {
+fn is_in_block(
```

Comment:
The comment here needs to be updated. Also, since we are no longer returning a boolean, maybe this should be named `validate_is_in_block()` or something like that?